public/js/app.js.LICENSE.txt

```diff
@@ -0,0 +1,67 @@
+/* istanbul ignore else -- @preserve */
```

Comment:
This is generated automatically by Mix when you enable versioning - it generates a summary of the licenses of your dependencies to help you ensure you are in compliance

Reply:
Which, speaking of which, we should probably decide on a license to set this repository under. [Leaving a codebase unlicensed is generally not great](https://choosealicense.com/no-permission/) especially if ZTL goes down the path of becoming a nonprofit

Reply:
LOL, never noticed that we didn't have one set...